app/src/main/java/com/e/salesapp/MainActivity.java

```diff
@@ -92,11 +104,21 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
 
         switch (item.getItemId()) {
             case R.id.action_cart:
-                Log.d("CART", "CLICKED");
+                Intent cartIntent = new Intent(this, InformationActivity.class);
+                startActivity(cartIntent);
                 break;
             default:
                 break;
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onClickCategory() {
+        FragmentManager manager = getSupportFragmentManager();
+        manager.beginTransaction()
+                .replace(R.id.container, new ProductListFragment())
```

Comment:
container is very generic name could please change the name for something more specific